ui/appui-react/src/appui-react/UiFramework.ts

```diff
@@ -657,10 +657,12 @@ export class UiFramework {
     return useGlobalStore.getState().viewState;
   }
 
+  /** @deprecated in 4.17.0. */
   public static getIsUiVisible() {
     return UiFramework.visibility.isUiVisible;
   }
 
+  /** @deprecated in 4.17.0. */
```

Comment:
Needs a replacement message.
```suggestion
  /** @deprecated in 4.17.0. Use {@link UiFramework.visibility.isUiVisible} instead. */
```

Reply:
Done.